dev/TabView/TabView.xaml

```diff
@@ -648,7 +689,7 @@
                             BorderThickness="{TemplateBinding BorderThickness}"
                             Control.IsTemplateFocusTarget="True"
                             Padding="{ThemeResource TabViewItemHeaderPadding}"
-                            CornerRadius="{Binding Source={ThemeResource OverlayCornerRadius}, Converter={StaticResource TopCornerRadiusFilterConverter}}"
+                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
```

Comment:
I think this was needed since we otherwise would have a gap in layout. Are we sure that this won't be an issue anymore?

Reply:
I'll check